library/core/src/iter/adapters/fuse.rs

```diff
@@ -181,6 +181,21 @@ where
     }
 }
 
+#[stable(feature = "default_iters", since = "CURRENT_RUSTC_VERSION")]
+impl<I: Default> Default for Fuse<I> {
```

Comment:
Pondering: I noticed that the other adapters require `I: Iterator` in addition to just `I: Default`.

Should this one require that too?

Reply:
Not sure, the bounds on the adapters might have been unintentional due to copypasta. Removing them would also be an option.

Reply:
Hello @rust-lang/libs-api folks!  This PR, as currently written, allows `<Fuse<u32>>::default()` but not `<Rev<u32>>::default()`.  That feels unintentional, so nominating for a decision on what the intent was here -- notably there wasn't an API summary in the FCP so it's hard to know for sure.

Do you want:
- The current state of the PR
- The adapters to be `Default` only when wrapping iterators
- The adapters to be `Default` when wrapping any inner type that's `Default`

Reply:
I think the third option would be best. Relaxing the bounds as much as possible seems consistent with how we don't have bounds on the types (only on the methods), and it would play a bit better with `#[derive(Default)]` when putting a `Adapter<T>` in a struct.

Nobody in today's libs-api meeting was opposed to the third option, but @dtolnay and @BurntSushi weren't available today. (Ping! Do you have any strong opinions on this?)

Reply:
latest commit removes the bounds